docs/analyzers/6.1/sitemap.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+         <loc>None</loc>
+         <lastmod>2022-11-23</lastmod>
+         <changefreq>daily</changefreq>
+    </url>
+</urlset>
```

Comment:
Do we actually need this? I mean, what's the point of a sitemap if it doesn't actually describe the pages on the site?
Why `<loc>None</loc>`?
Why `<changefreq>daily</changefreq>`? (we're not actually going to update the docs daily, at least I hope so!)

Reply:
(can't comment on binary files, so doing it here)
Why the `sitemap.xml.gz`?

Reply:
I do not have answers. Maybe there's some benefit in theory of having the lastmod and the change frequency (although calling this directory likely to change daily is _foolish_).

Reply:
Let's not worry too much about this. If mkdocs does it this way, there's probably a good reason, and since I don't know much about sitemaps, I'm inclined to trust it...